src/daemon/jsonrpc/api/mod.rs

```diff
@@ -745,10 +746,26 @@ impl RpcApi for RpcImpl {
         let rev_txs = vec![cancel_db_tx, emer_db_tx, unvault_emer_db_tx];
         db_update_presigned_txs(&db_path, &db_vault, rev_txs.clone(), secp_ctx)?;
         db_mark_securing_vault(&db_path, db_vault.id)?;
+
+        // If this made the Emergency fully signed and we are a stakeholder, share
+        // it with our watchtowers.
+        let emer_db_tx =
+            db_emer_transaction(&db_path, db_vault.id)?.ok_or(JsonRpcError::internal_error())?;
+        if !db_vault.emer_shared && emer_db_tx.psbt.unwrap_emer().is_finalizable(secp_ctx) {
+            if let Some(ref watchtowers) = revaultd.watchtowers {
```

Comment:
Oh wait, so are we ok with stakeholders not having watchtowers?

Reply:
Yes, i see no reason to not accept it and at least for now we need it